examples/lib/pods.rego

```diff
@@ -2,32 +2,25 @@ package lib.pods
 
 import data.lib.core
 
-pods[pod] {
-    lower(core.kind) == "statefulset"
-    pod = core.resource.spec.template
+pod = core.resource.spec.template {
+    pod_templates := ["daemonset","deployment","job","replicaset","replicationcontroller","statefulset"]
```

Comment:
would you guys be interested in supporting the OpenShift `DeploymentConfig` kind as well?

Reply:
I'm not entirely against the idea. I would have concerns about considering _every_ edge case and having the library spiral out of control--but just a single case isn't going to be that big of a problem. In a perfect scenario, we'd have an `openshift.rego` library that implemented `core` and added additional pod definitions (and other openshift specific definitions).

@jalseth , any thoughts on this one?

Reply:
I'd prefer if we added an `lib.openshift` that was a shim for `lib.core` with OpenShift specific items added. However, at this point, adding a shim just to match on one more Kind is probably excessive. We should revisit this when any other OpenShift items need to be added to the libraries.